docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: '2.1'
+version: '3.1'
```

Comment:
please remove this version bump. Seems like nothing needs docker-compose version 3.1 :smile: